download/index.md

```diff
@@ -3,26 +3,26 @@ layout: doc
 title: Download
```

Comment:
that whole section should be reviewed. 

Some examples still point to v3.1 and Activator is long gone. 

I will fix it in a second iteration.